### PR TITLE
Use nginx_buildpack instead of staticfile_buildpack

### PR DESCRIPTION
**Pull request by tlwr**
The staticfile_buildpack no longer supports custom nginx configuration, as such this repo is not in a deployable state.

The makefiles are a bit confusing and we can rely on the PaaS documentation instead.

We can rely on the manifest being a template, and using the `--var` features of CF push as well.

I've also updated the nginx configuration to include the correct HTTP version when proxying.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,139 +1,57 @@
 # Reliability Engineering IP Whitelisting Route Service
```
Comment (blairboy362) [inline README.md]:
```suggestion
# Reliability Engineering IP Safelisting Route Service
```

File: README.md [hunk 2]
```diff
@@ -1,139 +1,57 @@
...
-```sh
-CURRENT_APP=flask-example
-URL=$(cf curl /v2/apps/$(cf app --guid $CURRENT_APP)/summary | jq -r '.routes[0] | @uri "https://\(.host).\(.domain.name)"')
-curl $URL
-```
+You should log in using the Cloud Foundry CLI
+(https://docs.cloud.service.gov.uk/#setting-up-the-command-line).
 
-Copy `environment_samples.sh` to `environment.sh`. Edit it for cloud.gov so it looks something like this:
+For all actions you should always make sure you selected the space you intend
```
Comment (blairboy362) [inline README.md]:
```suggestion
For all actions you should always make sure you are targeting the correct space.
```

File: README.md [hunk 3]
```diff
@@ -1,139 +1,57 @@
...
+to target.
```
Comment (blairboy362) [inline README.md]:
```suggestion
```

**Review by blairboy362 — DISMISSED** [review]
Couple of very minor suggestions but not enough to stop a merge. LGTM. 👍

File: README.md (hunk 1 above)
Comment (tlwr) [reply README.md]: Renamed to `GOV.UK PaaS IP authentication route service`

File: README.md (hunk 2 above)
Comment (tlwr) [reply README.md]: Done in `e05ca23`

**Pull request merged**